static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -1,35 +1,90 @@
-import {useState} from 'react';
+import {useCallback, useEffect, useState} from 'react';
 import styled from '@emotion/styled';
 
+import {promptsCheck, promptsUpdate} from 'sentry/actionCreators/prompts';
+import Feature from 'sentry/components/acl/feature';
 import {Button} from 'sentry/components/button';
 import Card from 'sentry/components/card';
+import {openConfirmModal} from 'sentry/components/confirm';
+import {DropdownMenu, MenuItemProps} from 'sentry/components/dropdownMenu';
 import ExternalLink from 'sentry/components/links/externalLink';
 import {Tooltip} from 'sentry/components/tooltip';
-import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import {IconCommit, IconEllipsis, IconGithub, IconInfo, IconMail} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import {MissingMember, Organization} from 'sentry/types';
+import {promptIsDismissed} from 'sentry/utils/promptIsDismissed';
+import useApi from 'sentry/utils/useApi';
 import withOrganization from 'sentry/utils/withOrganization';
 
 type Props = {
   missingMembers: {integration: string; users: MissingMember[]};
-  onSendInvite: (email: string) => Promise<void>;
+  onSendInvite: (email: string) => void;
   organization: Organization;
 };
 
 export function InviteBanner({missingMembers, onSendInvite, organization}: Props) {
   // NOTE: this is currently used for Github only
-  // TODO(cathy): include snoozing, docs link
   const [sendingInvite, setSendingInvite] = useState<boolean>(false);
+  const [showBanner, setShowBanner] = useState<boolean>(false);
+
+  const api = useApi();
+  const integrationName = missingMembers?.integration;
+  const promptsFeature = integrationName + '_missing_members';
+
+  const snoozePrompt = useCallback(async () => {
+    await promptsUpdate(api, {
+      organizationId: organization.id,
+      feature: promptsFeature,
+      status: 'snoozed',
+    });
+
+    setShowBanner(false);
+  }, [api, organization, promptsFeature]);
+
+  useEffect(() => {
+    let isUnmounted = false;
+
+    promptsCheck(api, {
+      organizationId: organization.id,
+      feature: promptsFeature,
+    }).then(prompt => {
+      if (isUnmounted) {
+        return;
+      }
+
+      setShowBanner(!promptIsDismissed(prompt));
+    });
+
+    return () => {
+      isUnmounted = true;
+    };
```

Comment:
You shouldn't need to do this when using `useApi`. As long as [`persistInFlight`](https://github.com/getsentry/sentry/blob/ac9964f6535ae23f32c4ccf1a548626e42309e62/static/app/utils/useApi.tsx#L13-L21) is not enabled it will cancel the request on unmount and it will never resolve the promise.